src/bindings/python/src/pyopenvino/core/core.cpp

```diff
@@ -393,10 +393,17 @@ void regclass_Core(py::module m) {
 
     cls.def(
         "read_model",
-        (std::shared_ptr<ov::Model>(ov::Core::*)(const std::string&, const std::string&) const) & ov::Core::read_model,
-        py::call_guard<py::gil_scoped_release>(),
+        [](ov::Core& self,
+           const std::string& model_path,
+           const std::string& weight_path,
+           const std::map<std::string, py::object>& properties) {
+            const auto any_map = Common::utils::properties_to_any_map(properties);
+            py::gil_scoped_release release;
```

Comment:
Sure, but how I can test from py API that local config is applied?

Reply:
> Sure, but how I can test from py API that local config is applied?

you can try to remove file on Windows while ov::Model is here and it should fail, because file is opened and cannot be removed.

Reply:
Test added